src/PhpTypes.php

```diff
@@ -73,16 +73,16 @@ protected function init(): void {
 
     private function initName(): PhpName {
         $name = Html::queryFirstValue($this->xpath(), '//*[@class="classsynopsisinfo"]//strong[@class="classname"]', null, true);
+        if ($name === null) {
+            // e.g. class.throwable.html
+            $name = Html::queryFirstValue($this->xpath(), '//*[@class="classsynopsisinfo"]//strong[@class="interfacename"]', null, true);
```

Comment:
Just an idea - cannot it also be something like `traitname`? Or `enumname`? :sweat_smile:

Reply:
Cool, thanks for the verification.